README.md

```diff
@@ -55,9 +55,8 @@ Create an index.html file (see [docs](/docs/root-application.md#indexhtml-file)
 Create the root application (see [docs](/docs/root-application.md) for more detail).
 ```js
 // src/main.js
-import { declareChildApplication, setLoader, start } from "single-spa";
-setLoader(SystemJS);
-declareChildApplication('/apps/app1/app1.js', () => window.location.hash === '');
+import { declareChildApplication, start } from "single-spa";
+declareChildApplication('/apps/app1/app1.js', () => System.import('/apps/app1/app1.js'), () => window.location.hash === '');
```

Comment:
Why does the api need the `name` and the `doLoad`? Couldn't it just be `doLoad` and `activeWhen`?